css/main.css

```diff
@@ -72,5 +77,1037 @@ select {
     scroll-behavior: auto !important;
   }
 }
+@font-face {
+  font-family: "cocogoose";
+  src: url(../assets/fonts/cocogoose/cocogoose-webfont.woff);
+  font-weight: 900;
+}
+@font-face {
+  font-family: "cocogoose";
+  src: url(../assets/fonts/cocogoose/cocogoose-Italic.ttf);
+  font-style: italic;
+}
+html {
+  font-size: 62.5%;
+  font-family: "lato", sans-serif;
+}
+
+body {
+  font-size: 1.6rem;
+}
+
+.mw-container {
+  width: 100%;
+  max-width: 96rem;
+  margin-left: auto;
+  margin-right: auto;
+  display: inherit;
+  flex-direction: inherit;
+  align-content: inherit;
+  justify-content: inherit;
+  align-items: inherit;
+  gap: inherit;
+  column-gap: inherit;
+}
+
+@media (min-width: 768px) {
+  .only-mobile {
+    display: none !important;
+  }
+}
```

Comment:
- [ ] There are about 40 appearances of media queries in your CSS file. Kindly group them at the bottom of the file for seamless readability. Do the same for other queries.

Reply:
Hello, can you please give more insight in this topic. I was using SASS in order to write the CSS file and I thought I was doing it correctly in the SASS files. I haven't been able to find any good information about how to write media queries in SASS that get compiled at the end of the CSS file.

Also, I know we were supposed to use #EC5242, but we were also encouraged to personalized our site without going to far away from the original design layout, colors, etc. That's why I thought that the green shade I used was more appropriate for a Camping event. Every other was respected.

>Choose a topic first and then design your webpage, keeping your topic in mind while you design it so that your page design works well for the topic you choose.

Reply:
@D4nielJ There are a lot of ways you can use media queries. You can store them into variables, you can create a different file for different media sizes, and import them into your main file.